src/App/Action/Backend/Account/FetchSubscriptionFeaturesAction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\App\Action\Backend\Account;
+
+use MyParcelNL\Pdk\Account\Repository\SubscriptionFeaturesRepository;
+use MyParcelNL\Pdk\Api\Response\JsonResponse;
+use MyParcelNL\Pdk\App\Action\Contract\ActionInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class FetchSubscriptionFeaturesAction implements ActionInterface
+{
+    /**
+     * @var \MyParcelNL\Pdk\Account\Repository\SubscriptionFeaturesRepository
+     */
+    private $subscriptionFeaturesRepository;
+
+    /**
+     * @param  \MyParcelNL\Pdk\Account\Repository\SubscriptionFeaturesRepository $subscriptionFeaturesRepository
+     */
+    public function __construct(
+        SubscriptionFeaturesRepository $subscriptionFeaturesRepository
+    ) {
+        $this->subscriptionFeaturesRepository = $subscriptionFeaturesRepository;
+    }
+
+    /**
+     * @param  \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function handle(Request $request): Response
+    {
+        return new JsonResponse($this->subscriptionFeaturesRepository->getSubscriptionFeatures());
```

Comment:
de wrapper property van de request mist hier, dit zou de frontend nu niet kunnen gebruiken mochten we dat willen